### PR TITLE
Add asymmetric cryptography slides

### PR DESCRIPTION
**Pull request by Rothj**
# Description

* Created the first version of the slides on asymmetric cryptography
* Including two graphics
* Added Rivest et al. in reference file

## Type of Change

- [x] Major feature (New files or assets, large changes to files)

# Checklist:

- [x] I have performed a self-review of my own changes
- [x] I have made sure that any changed LaTeX files compile properly
- [x] I have assigned at least one reviewer

### CODE REVIEW

File: slides/Asymmetric Cryptography.tex [hunk 1]
```diff
@@ -0,0 +1,229 @@
+% Choose one to switch betweeen slides and handout
+\documentclass[]{beamer}
+%\documentclass[handout]{beamer}
+
+% Video Meta Data
+\title{Bitcoin, Blockchain and Cryptoassets}
+\subtitle{Intro: Welcome to the Course}
```
Comment (dariothuerkauf) [inline slides/Asymmetric Cryptography.tex:7]:
```suggestion
\subtitle{Asymmetric Cryptography}
```

File: slides/Asymmetric Cryptography.tex [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+	\end{itemize}
+	\uncover<3->{
+		\begin{figure}
+			\centering
+			\includegraphics[width = 8cm]{../assets/images/cover_RSA}
+		\end{figure}
+	}
+\end{frame}
+%%%
+
```
Comment (dariothuerkauf) [inline slides/Asymmetric Cryptography.tex:115]: What do you think about adding the two use cases "Privacy" and "Integrity/Authenticity" of asymmetric cryptography in this section? There's a nice Figure in the MIT book on that.

File: slides/Asymmetric Cryptography.tex [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+%%%
+\begin{frame}{RSA}
+	Properties of RSA
+	\begin{itemize}
+		\item<1->[(a)] Alice creates a \textit{public key} with which everyone can encrypt a message. The encryption with the public key is a one-way function and therefore not revertable
+		\item<2->[(b)] Only Alice is able to decrypt the message with her \textit{private key} and revert the encryption process
+	\end{itemize}
+\end{frame}
+%%%
+
```
Comment (dariothuerkauf) [inline slides/Asymmetric Cryptography.tex:125]: What do you think about adding the two use cases "Privacy" and "Integrity/Authenticity" of asymmetric cryptography in this section? There's a nice figure in "Bitcoin, Blockchain & Cryptoassets" on that.

**Comment by dariothuerkauf** [comment]
Good Job! All the important examples are included and you maintained the historical background, which I really liked in last year's lecture. Eventually, we should also define in the Style Guidelines how important terms such as "private key" etc. should be highlighted (italics or focus colour).

File: slides/Asymmetric Cryptography.tex (hunk 1 above)
Comment (Rothj) [reply slides/Asymmetric Cryptography.tex:7]: Good catch! Thanks! Totally overlooked that one!

**Review by Chainomics — APPROVED** [review]

**Pull request merged**